vehicle-python/vehicle/__init__.py

```diff
@@ -168,6 +170,25 @@ def _translate_maximum(self, contents: Dict[Any, Any]) -> str:
         loss_2 = self._translate_expression(contents[1])
         return f"max({loss_1}, {loss_2})"
 
+    # NOT WORKING CURRENTLY
```

Comment:
If this isn't working currently, wouldn't it be better to comment this code out and throw an error? Rather than lulling the user into thinking it's working?